js/customize-snapshots.js

```diff
@@ -980,22 +980,32 @@
 		 * @returns {string} Current datetime string.
 		 */
 		getCurrentTime: function getCurrentTime() {
-			var snapshot = this,
-				currentDate = new Date( snapshot.data.initialServerDate ),
-				currentTimestamp = snapshot.dateValueOf(),
-				timestampDifferential;
-
+			var snapshot = this, currentDate, currentTimestamp, timestampDifferential;
+			currentTimestamp = ( new Date() ).valueOf();
+			currentDate = snapshot.parsePostDate( snapshot.data.initialServerDate );
 			timestampDifferential = currentTimestamp - snapshot.data.initialClientTimestamp;
 			timestampDifferential += snapshot.data.initialClientTimestamp - snapshot.data.initialServerTimestamp;
 			currentDate.setTime( currentDate.getTime() + timestampDifferential );
-
 			return snapshot.formatDate( currentDate );
 		},
 
 		/**
+		 * Parse post date string in YYYY-MM-DD HH:MM:SS format (local timezone).
+		 *
+		 * @param {string} postDate Post date string.
+		 * @returns {Date} Parsed date.
+		 */
+		parsePostDate: function parsePostDate( postDate ) {
+			var dateParts = _.map( postDate.split( /\D/ ), function( datePart ) {
+				return parseInt( datePart, 10 );
+			} );
+			return new Date( dateParts[0], dateParts[1] - 1, dateParts[2], dateParts[3], dateParts[4], dateParts[5] ); // eslint-disable-line no-magic-numbers
+		},
```

Comment:
I wish we could pass an array in the `Date` object, but one day we will be able to do it using spread syntax, like this, ( though it works in chrome ) 😀
```
return new Date(... dateParts);
```